### PR TITLE
add gibberish-aes@1.0.0 w/ git autoupdate

### PR DESCRIPTION
**Pull request by iwdmb**
#8741

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/mdp/gibberish-aes
- GitHub:
  - Count of stars: 415
  - Count of watchers: 36
  - Count of forks: 93
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/gibberish-aes/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Gibberish-AES",
```
Comment (pvnr0082t) [inline ajax/libs/gibberish-aes/package.json]: It should be the same as the folder name, so let's use `gibberish-aes` here.

**Comment by pvnr0082t** [comment]
Please add `keywords` field in package.json. Thanks.

**Comment by pvnr0082t** [comment]
@iwdmb Because the naming scheme of the repo contains the version. Could you please discuss if it's possible to change the naming scheme with the author?

**Comment by pvnr0082t** [comment]
If the repo has been published on npm, we prefer to use npm auto-update config. 
BTW, there is no any valid git tag of this repo. ref: https://github.com/mdp/gibberish-aes/releases

**Comment by pvnr0082t** [comment]
Please add `close #8741, cc @[author]` in the commit message. Thank you.

**Comment by Piicksarn** [comment]
@iwdmb, any update?

**Comment by iwdmb** [comment]
Moved to #9810

**Pull request closed without merging**